besca/Rlibs.R

```diff
@@ -0,0 +1,17 @@
+args = commandArgs(trailingOnly=TRUE)
+libloc <-  args[1]
+
+dir.create(libloc)
+
+.libPaths(libloc)
+if (!require("devtools")) install.packages("pacman", lib = libloc)
```

Comment:
Some copy paste issue? 

if (!require("devtools")) install.packages(**"devtools"**, lib = libloc)

Same from **withr** and **Matrix** ?

Reply:
Wops, yes! my bad, sorry.